### ISSUE TITLE
Design calibration routine for DRRP without removing optics

### ISSUE THREAD

**Issue by Jashcraf** [UCSB-Exoplanet-Polarimetry-Lab/derp_control#14]
It would be very useful to be able to take Mueller Matrix measurements without removing optics. To do so - we need to come up with a calibration method and test it! A few ideas:
- Start with PSG polarizer and Walloston in a crossed configuration (with one of the channels at least)
- Rotate waveplates until a similar null is reached. This means we are near the fast or slow axis of both of the waveplates

**Comment by Jashcraf** [comment]
@wcmelby has already done this to some degree

**Comment by Jashcraf** [comment]
Ronald and I tried out a method of doing this, see the 10/16 entry in #4

**Comment by wcmelby** [comment]
I finished the demo of my method. It requires some functions from katsu but I think it would still be suitable to include here. I'm curious to see how it compares to other methods as well.

The calibration procedure I've used is relatively fast; it needs only one set of measurements per wavelength and doesn't require removing components. In practice, I've been able to correct the error in a Mueller matrix by an order of magnitude. This demo also shows how the correction could be several orders of magnitude better if we could properly normalize the laser intensity. 

I also pointed out areas for improvement in terms of accounting for more error terms and exploring the degeneracies between them. 

My account needs permission to push these changes on Github if possible.
